tequilapi/endpoints/location.go

```diff
@@ -0,0 +1,52 @@
+package endpoints
+
+import (
+	"github.com/julienschmidt/httprouter"
+	"github.com/mysterium/node/location"
+	"github.com/mysterium/node/tequilapi/utils"
+	"net/http"
+)
+
+// LocationEndpoint struct represents /location resource and it's subresources
+type LocationEndpoint struct {
+	locationDetector location.Detector
+	locationCache location.Cache
+}
+
+// NewLocationEndpoint creates and returns location endpoint
+func NewLocationEndpoint(locationDetector location.Detector, locationCache location.Cache) *LocationEndpoint {
+	return &LocationEndpoint{
+		locationDetector: locationDetector,
+		locationCache: locationCache,
+	}
+}
+
+// GetLocation responds with original and current countries
+func (ce *LocationEndpoint) GetLocation(writer http.ResponseWriter, request *http.Request, params httprouter.Params) {
+	originalLocation, err := ce.locationCache.Get()
```

Comment:
I would make it '(le *LocationEndpoint)', not '(ce *LocationEndpoint)'

Reply:
> If we failed to get location (original) on startup do we always display that error?

In case only one of original and current ip is not available, it makes sense to return what we have. I.e.
```js
{
  original: null,
  current: "1.1.1.1"
}
```
or
```js
{
  original: "1.1.1.1",
  current: null
}

Reply:
@tadovas changed logic where locationCache.Get() no longer returns error so the answer is no, we display error on startup only